index.js

```diff
@@ -1,10 +1,52 @@
 var express = require('express');
 var app = express();
+var bodyParser = require('body-parser');
 var port = 8000;
 
 // write the API here
 
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
 
+app.get('/api', function(req, res) {
+	//res.set('Content-Type', /json/);
+	//res.sent({thoughts: 'First Test'}
+	res.send('Welcome to my API!');
```

Comment:
This is kinda my fault... Before, the tests where not looking for json. Now they are, for all the routes.